andela/mmdp_site
# MMDP-464 Implement manage beneficiary type dropdown

**What does this PR do?**

This PR implements CRUD operations for the  beneficiary type dropdown

**Description of Task to be completed?**

As an administrator, I want to be able to manage options in the 'Beneficiary Type' drop down

**How should this be manually tested?**

Git clone https://github.com/andela/mmdp_site
Check out  ft-beneficiary-type-MMDP-464 branch using git.
Run yarn run: server to start the server.

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-464](https://viisaus.atlassian.net/browse/MMDP-464)
